va/va.h

```diff
@@ -492,6 +492,20 @@ typedef enum
      */
     VAConfigAttribEncMacroblockInfo     = 16,
     /**
+     * \brief Maximum picture width. Read-only.
+     *
+     * This attribute determines the maximum picture width the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureWidth     = 18,
+    /**
+     * \brief Maximum picture height. Read-only.
+     *
+     * This attribute determines the maximum picture height the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureHeight    = 19,
+    /**
```

Comment:
User must specify a ***config*** when calling vaQuerySurfaceAttributes(), so VASurfaceAttribMaxWidth/VASurfaceAttribMaxHeight is valid only for the ***given config***, e.g. for VAProfileH264Main/VAEntrypointVLD, VASurfaceAttribMaxWidth/VASurfaceAttribMaxHeight may be 1920/1080, but for VAProfileNone/VAEntrypointVideoProc, VASurfaceAttribMaxWidth/VASurfaceAttribMaxHeight  may be 4K/4K.  The API doesn't require VASurfaceAttribMaxWidth/VASurfaceAttribMaxHeight must be same for all ***config***